src/app/core/components/event-card/event-card.component.scss

```diff
@@ -0,0 +1,154 @@
+@import 'src/styles-variables.scss';
+
+.event-card {
```

Comment:
At first, I thought these 4 inner components need to share some styles/template/logic. It turns out it hasn't.

Since CSS styles of these 4 type of cards are completely independent with each others, and their templates HTML code can be grouped by similarity into 3 groups (pocket/list-small, list-regular, and full-card), It should not be hard for us to refactor this single event-card component to 3 components.